lib_brewing/data/gm4_brewing-1.0/functions/prep_finish_brew.mcfunction

```diff
@@ -0,0 +1,12 @@
+# @s = brewing stand that finished brewing
+# run from check_brew
+
+# mark as finished
+tag @s remove gm4_brewing_tick
+tag @s add gm4_brewing_finished
+
+# store contents of brewing stand
+data modify entity @s data.Items set from block ~ ~ ~ Items
```

Comment:
This should be changed to `data.gm4_lib_brewing.<key>`. I think a more descriptive name such as `data.gm4_lib_brewing.previous_items` might make for a cleaner look.